packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -589,6 +590,16 @@ export default class TestConfiguration {
     })
   }
 
+  async duplicateApp(appId: string, fields: object) {
+    return context.doInTenant(
+      this.tenantId!,
+      async () =>
+        await this._req(appController.duplicateApp, fields, {
+          appId,
+        })
+    )
+  }
```

Comment:
We're in the process of migrating out API calls from the `TestConfiguration` class and into the relevant API classes under `packages/server/src/tests/utilities/api`. Could you move this into the `ApplicationAPI` class, please? 🙏